### PR TITLE
Varun/cli tool

### PR DESCRIPTION
**Pull request by vsn-hub**
This directory contains the cli-tool that will be used to integrate with github. Work in progress. Has a bit of Readme if anyone wants to try it out.

### CODE REVIEW

File: cli-tool/my_tool.py [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+def cli():
+    pass
+
+
+@cli.command()
+@click.option('-d', '--dest', type=str, help='Path to Swagger File')
+def send(dest):
+    #click.echo(f'Shit {dest}')
+    
+    API_ENDPOINT = "https://o8cnchwjji.execute-api.ap-southeast-2.amazonaws.com/v1/post_json"
```
Comment (thefishua) [inline cli-tool/my_tool.py:14]: I believe this API endpoint won't necessarily be the same in production so you will need to create an environment variable from terraform

File: cli-tool/mytool.egg-info/PKG-INFO [hunk 2]
```diff
@@ -0,0 +1,13 @@
+Metadata-Version: 2.1
```
Comment (thefishua) [inline cli-tool/mytool.egg-info/PKG-INFO:1]: What is this file ?

**Comment by thefishua** [comment]
Have made so comments on the code but overall looks good [comment 1](https://github.com/stevensnicholas/comp9447-team2/pull/44/files#r920624512) [comment 2](https://github.com/stevensnicholas/comp9447-team2/pull/44/files#r920624865)

**Comment by The-TT-Hacker** [comment]
the CLI took should be in golang, python in not appropriate for this. golang will build you a static binary people can download instead of a bunch of script files. We already have typescript and golang you shouldnt be adding a new programming language unless its 100% justifiable since it adds a lot of overhead in terms of setting up tooling to test it, lint it etc.

File: cli-tool/my_tool.py (hunk 1 above)
Comment (vsn-hub) [reply cli-tool/my_tool.py:14]: sure will do that

**Comment by vsn-hub** [comment]
Makes sense I'll switch it over to golang

File: cli-tool/my_tool.py (hunk 1 above)
Comment (MrC1rcle) [reply cli-tool/my_tool.py:14]: what does the api endpoint do here?

**Comment by The-TT-Hacker** [comment]
@vsn-hub golang is super easy, expecially for this use case. Just add a new directory and main function in the cmd directory

File: cli-tool/my_tool.py (hunk 1 above)
Comment (The-TT-Hacker) [reply cli-tool/my_tool.py:14]: make it configurable with a command line or environment variable

Comment (MrC1rcle) [reply cli-tool/my_tool.py:14]: ok got that

**Comment by vsn-hub** [comment]
> @vsn-hub golang is super easy, expecially for this use case. Just add a new directory and main function in the cmd directory

will do that thanks

**Pull request closed without merging**